CMakeLists.txt

```diff
@@ -167,6 +167,7 @@ set(VLLM_EXT_SRC
   "csrc/layernorm_kernels.cu"
   "csrc/quantization/squeezellm/quant_cuda_kernel.cu"
   "csrc/quantization/gptq/q_gemm.cu"
+  "csrc/quantization/compressed_tensors/int8_quant_kernels.cu"
```

Comment:
It's a bit unclear to me about the name `compressed_tensors`.  I suppose this is the official method name of SparseML? Then can we just use `sparseml` here?

Reply:
`compressed-tensors` is the name of the package responsible for saving quantized and sparse models

So the flow is:
- use SparseML to apply quantization / sparsity
- save model to `safetensors` with a `compressed-tensors` config
- load + run in vllm